.github/workflows/release.yaml

```
@@ -0,0 +1,82 @@
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_REPO: seanconroy2021/golden-container
+  IMAGE_TAGS: latest 
+
+jobs:
+  build:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Needed for signing the container image with GitHub OIDC Token
+      write-all
```

Bit much, perhaps we can narrow this to:

```suggestion
      # Needed for signing the container image with GitHub OIDC Token
      id-token: write
      # Needed to push to ghcr.io registry
      packages: write
```